src/app/serverViewState.ts

```diff
@@ -343,7 +343,7 @@ export class ServerViewState {
         const server = new MattermostServer({name: 'temp', url: parsedURL.toString()}, false);
         const serverInfo = new ServerInfo(server);
         try {
-            const remoteInfo = await serverInfo.fetchRemoteInfo();
```

Comment:
Nit: this variable is no longer storing `remoteInfo`, but `configData`, so it should be renamed.

Similar in the many other places this function is called on this file.

Reply:
It actually is still storing the `RemoteInfo` type, it's just not necessarily complete at this point. So I think the naming is okay.